### PR TITLE
Setup component for core functionality and UI of the application.

### PR DESCRIPTION
**Pull request by jodavid444**
# In This Pull Request

- Create a new component directory
- Create a new calculator.js file
- Create react component that matches the given UI design.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+import ReactDOM from 'react-dom/client';
+import './index.css';
+import Calculator from './components/calculator';
+
+const root = ReactDOM.createRoot(document.getElementById('root'));
+root.render(
+  <React.StrictMode>
+    <Calculator />
+  </React.StrictMode>,
+);
```
Comment (emmiiorji) [inline src/index.js:11]: It's good that you have removed the boilerplate files by the CRA. By convention, App.js and App.css make the main component and therefore should not be deleted, only the content that is not necessary should be removed. Kindly include them.

**Review by emmiiorji — CHANGES_REQUESTED** [review]
Hi @jodavid444, 
# Status: Changes Required ♻️

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

BbcanBbcan3GIF (2)

## Good Highlights✔️

- Linter checks are passing
- Good commit messages
- Proper use of Git flow
- Good design


## Required Changes ♻️
_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js (hunk 1 above)
Comment (jodavid444) [reply src/index.js:11]: I have added both files to the project.

**Review by rotimiazeez — APPROVED** [review]
Hi @jodavid444 👏🏽, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 
- Good communication with the CR 👍🏼
- No linter error ✅️
- Descriptive README 👍🏼

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code. Some of them were simply missed by the previous reviewer, and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me(@rotimiazeez) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**